app/mailers/spree/user_mailer.rb

```diff
@@ -8,5 +10,34 @@ def reset_password_instructions(user)
           :subject => Spree::Config[:site_name] + ' ' + 
             I18n.t(:subject, :scope => [:devise, :mailer, :reset_password_instructions]))
     end
+
+    def signup_confirmation(user)
+      @user = user
+      I18n.with_locale valid_locale(@user) do
+        mail(to: user.email, from: from_address,
+             subject: t(:welcome_to) + Spree::Config[:site_name])
+      end
+    end
+
+    # Overriding `Spree::UserMailer.confirmation_instructions` which is
```

Comment:
Hm, I expected this to replace something. Did spree_auth_devise stop overriding this? Does this comment need adjusting?

Reply:
I'll have a close look at this.

Reply:
this method is not there in v2.0, only introduced in v2.4 [here](https://github.com/openfoodfoundation/spree_auth_devise/commit/c25cfa1812438d41332f7e1545ea56fc1ce3c01d)
Anyway, I added a new commit to fix the comments, see [new commit](https://github.com/openfoodfoundation/openfoodnetwork/pull/3700/commits/942824cd74ca4528d2f58c507352bddc6a94d49c).